README.md

```diff
@@ -1,5 +1,47 @@
 # Ansible
-Ansible installer for h5radar application
+Ansible installer for h5radar application. Demo available at https://app.h5radar.com.
+
+# How to install application
+There are a few way to install application. The main difference related to SSL 
+certification: self signed, let's encrypt or external files.
+
+# Self signed certificates
+* copy inventory/development/ files to inventory/production
```

Comment:
@borismaslov this section should be verified. A new customer should be able to setup application step by step.

Reply:
Issue (https://github.com/h5radar/ansible/issues/27) has been created